src/components/Play Button/styles.tsx

```diff
@@ -1,14 +1,14 @@
 import styled from "styled-components";
 import colors from "../../design/colors";
 
-export const StyledButton = styled.button`
+export const PlayStopIcon = styled.div`
   color: white;
   width: 20px;
   height: 20px;
-  background-color: ${({ isActive }) =>
+  background-color: ${({isActive}) =>
     isActive ? colors.mainBlue : colors.white};
   border: solid 3.5px ${colors.mainBlue};
-  margin: ${({ isActive }) => (isActive ? "0" : "5px")};
+  margin: 0 auto;
```

Comment:
Had to change this from a `styled.button` to a `styled.div` because I was seeing an error about a button being nested in a button. 

Changing it to a div made the default margins change so I made it centered.